CMakeLists.txt

```diff
@@ -60,8 +62,10 @@ find_package(Eigen3 REQUIRED)
 include_directories(${EIGEN3_INCLUDE_DIR})
 
 if (tricktrack_logger)
-  find_package(spdlog REQUIRED)
-  include_directories(${SPDLOG_INCLUDE_DIR})
+  find_package(spdlog)
+  if (NOT spdlog_FOUND) 
+    include_directories(include/spdlog)
+  endif()
```

Comment:
@vvolkl Would you need to put `include_directories(${SPDLOG_INCLUDE_DIR})` back again if `spdlog_FOUND` ?

Reply:
Ah, completely right, I'll patch this!